client/locales/de.json

```diff
@@ -5,22 +5,22 @@
         "cancel": "Abbrechen",
         "overwrite": "Überschreiben",
         "continue": "Weiter",
-        "dismiss": "Abbrechen",
+        "dismiss": "Verwerfen",
         "prompt": "Hinweis",
         "warning": "Warnung",
         "attention": "Achtung!",
         "confirm": "Bestätigen",
         "close": "Schließen",
         "save": "Speichern",
         "cancelled": "Abgebrochen",
-        "enabled": "Eingeschaltet",
+        "enabled": "Aktiviert",
         "commandOK": "Befehl erfolgreich übertragen!",
         "commandFailed": "Es ist ein Fehler aufgetreten:",
         "connectionError": "Verbindungsfehler",
         "m2": "m²",
         "hours": "Stunde",
         "hours_plural": "Stunden",
-        "months": ["Jan","Feb","Mär","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dez"],
+        "months": ["Jan","Feb","Mär","Apr","Mai","Jun","Jul","Aug","Sep","Oct","Nov","Dez"],
```

Comment:
"months": ["Jan","Feb","Mär","Apr","Mai","Jun","Jul","Aug","Sep","Okt","Nov","Dez"],

=> Okt

Reply:
THX, I have read too much english in the last time.